Packs/FeedCyrenThreatInDepth/Integrations/CyrenThreatInDepth/CyrenThreatInDepth.yml

```diff
@@ -0,0 +1,137 @@
+commonfields:
+  id: CyrenThreatInDepth
+  version: -1
+name: CyrenThreatInDepth
+display: Cyren Threat InDepth Threat Intelligence
+category: Data Enrichment & Threat Intelligence
+description: Threat InDepth's actionable and contextualized intelligence helps
+  enterprises improve their threat detection and response by providing unprecedented
+  visibility into new email-borne security threats faster than other security vendors.
+fromversion: 5.5.0
+configuration:
+- display: Indicator Reputation
+  name: feedReputation
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+- display: Source Reliability
+  name: feedReliability
+  defaultvalue: F - Reliability cannot be judged
+  type: 15
+  required: true
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  additionalinfo: Reliability of the source providing the intelligence data
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: indicatorType
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- display: ""
+  name: feedExpirationInterval
+  defaultvalue: "20160"
+  type: 1
+  required: false
```

Comment:
yes this is an incremental feed, but what does the parameter do in the end?

Reply:
In a regular feed the state is managed inside demisto, meaning that only when there are new or modified indicators we perform mapping and saving.
In an incremental feed, the state is managed in an external source, meaning that each time the feed is called, we map and save all the incoming indicators, since if the external source gave them, it means that there is something new or modified.

Reply:
added the parameter, I assume it is intended for the system using the integration not for our integration code to use directly...